### PR TITLE
Fix limitRange flake for deleting namespaces

### LINKED ISSUE #11094: LimitRange defaults flake
```
• Failure in Spec Teardown (AfterEach) [300.360 seconds]
[k8s.io] LimitRange
/data/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/framework.go:793
  should create a LimitRange with defaults and ensure pod has those defaults applied. [AfterEach]
  /data/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/limit_range.go:102

  Sep 26 03:54:52.649: Couldn't delete ns: "e2e-tests-limitrange-c4yb1": namespace e2e-tests-limitrange-c4yb1 was not deleted with limit: timed out waiting for the condition, namespace is empty but is not yet removed (&errors.errorString{s:"namespace e2e-tests-limitrange-c4yb1 was not deleted with limit: timed out waiting for the condition, namespace is empty but is not yet removed"})

  /data/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/framework.go:338
```

as seen in:
https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_conformance/6466/consoleFull

### PR DESCRIPTION
**Pull request by gnufied**
Increase the timeout we wait for namespace to be deleted before
giving up.

Fixes #11094

### CODE REVIEW

**Comment by gnufied** [comment]
cc @childsb

**Comment by gnufied** [comment]
After discussing this @childsb  this sounds like something we can push in upstream. Going to do that.

**Comment by gnufied** [comment]
@derekwaynecarr  PTAL.

File: vendor/k8s.io/kubernetes/test/e2e/limit_range.go [hunk 1]
```diff
@@ -29,6 +30,7 @@ import (
 
 var _ = framework.KubeDescribe("LimitRange", func() {
 	f := framework.NewDefaultFramework("limitrange")
+	f.NamespaceDeletionTimeout = 10 * time.Minute
```
Comment (liggitt) [inline vendor/k8s.io/kubernetes/test/e2e/limit_range.go]: this seems excessive... what actual times were we seeing?

**Comment by gnufied** [comment]
Amended the commit. But because there is no upstream PR, there is nothing to put there.  @liggitt PTAL

**Comment by liggitt** [comment]
> But because there is no upstream PR, there is nothing to put there

if the fix is needed, it should also be needed upstream, so open an upstream PR and reference the number here

**Comment by gnufied** [comment]
@liggitt See the comment by @derekwaynecarr here https://github.com/openshift/origin/issues/11094#issuecomment-253816354

**Comment by gnufied** [comment]
Closing this in favour of #11627

**Pull request closed without merging**